taskt/Core/Automation/Commands/Data Commands/GetTextLengthCommand.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+using System.Xml.Serialization;
+using taskt.Core.Automation.Attributes.ClassAttributes;
+using taskt.Core.Automation.Attributes.PropertyAttributes;
+using taskt.Core.Automation.Engine;
+using taskt.Core.Utilities.CommonUtilities;
+using taskt.UI.CustomControls;
+using taskt.UI.Forms;
+
+namespace taskt.Core.Automation.Commands
+{
+    [Serializable]
+    [Group("Data Commands")]
+    [Description("This command returns the length of a string.")]
+    public class GetTextLengthCommand : ScriptCommand
+    {
+        [XmlAttribute]
+        [PropertyDescription("Text Data")]
+        [InputSpecification("Provide a variable or text value.")]
+        [SampleUsage("Hello World || {vStringVariable}")]
+        [Remarks("Input data other than string (text) type will cause error.")]
```

Comment:
Change to "Providing data of a type other than a 'String' will result in an error."

Reply:
done